### PR TITLE
docs: fix image links in `performance.md`

### PR DESCRIPTION
**Pull request by RaisinTen**
#### Description of Change

Fixes https://github.com/electron/electron/issues/29580

Signed-off-by: Darshan Sen <raisinten@gmail.com>

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/master/docs/development/testing.md)
- [x] relevant documentation is changed or added
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: docs/tutorial/performance.md [hunk 1]
```diff
@@ -120,9 +120,9 @@ file in the directory you executed it in. Both files can be analyzed using
 the Chrome Developer Tools, using the `Performance` and `Memory` tabs
 respectively.
 
-![performance-cpu-prof]
+![Performance CPU Profile](../images/performance-cpu-prof.png)
```
Comment (dsanders11) [inline docs/tutorial/performance.md:123]:
```suggestion
![Performance CPU Profile][performance-cpu-prof]
```

File: docs/tutorial/performance.md [hunk 2]
```diff
@@ -120,9 +120,9 @@ file in the directory you executed it in. Both files can be analyzed using
...
 
-![performance-heap-prof]
+![Performance Heap Memory Profile](../images/performance-heap-prof.png)
```
Comment (dsanders11) [inline docs/tutorial/performance.md:125]:
```suggestion
![Performance Heap Memory Profile][performance-heap-prof]
```

**Review by dsanders11 — CHANGES_REQUESTED** [review]
@RaisinTen, nice catch. I've gone ahead and made some suggested changes. The URLs are already at the bottom of the doc as [reference-style links](https://www.markdownguide.org/basic-syntax/#reference-style-links), it was just the usage of the links for the images which had the wrong syntax.

**Review by mariasandra6 — APPROVED** [review]

**Comment by RaisinTen** [comment]
@dsanders11 Thanks for the suggestion. Updated, PTAL. :)

**Review by dsanders11 — APPROVED** [review]

**Review by codebytere — APPROVED** [review]

**Pull request merged**